apps/cyberstorm-remix/app/root.tsx

```diff
@@ -51,10 +28,12 @@ import {
 } from "cyberstorm/security/publicEnvVariables";
 import { StorageManager } from "@thunderstore/ts-api-react/src/storage";
 import { Route } from "./+types/root";
+import { RootLayout } from "./rootRoute/rootLayout";
 
 // REMIX TODO: https://remix.run/docs/en/main/route/links
 // export const links: LinksFunction = () => [{ rel: "stylesheet", href: styles }];
 
+/* Declaring window types that are present, because of certain parts of the application */
```

Comment:
Minor: I find this comment confusing. Certain parts of the application what? Expect the types to be declared?

Question: would it make sense to define the nitroAds part in `nitroAds.tsx`  to compartmentalize ads related stuff in that file?